### PR TITLE
tidb-in-kubernetes: refine tidb in k8s document

### PR DESCRIPTION
**Pull request by tennix**
### What is changed, added or deleted? 

TiDB in Kubernetes documents have been refined:
* The placeholder names are unified using lower case words separated by a dash.
* Adds descriptions for `chart-version` and `release-name`.
* Fix some typos and unclear descriptions.

### What is the related PR or file link(s)? 

None

### Which version does your change affect? 

3.0 and dev.

### Checklist 

- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by lilin90** [comment]
@tennix Please check the CI feedback and fix the format.

File: dev/tidb-in-kubernetes/scale-in-kubernetes.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
 > **注意：**
 >
 > - PD、TiKV 组件在滚动升级的过程中不会触发扩缩容操作。
-> - PD、TiKV 组件在缩容过程中都会调用接口下线正在删除的 PD、TiKV 组件在下线时会涉及到数据迁移的操作，所以会消耗比较长的时间。
+> - TiKV 组件在缩容过程中会调用 PD 接口将对应 TiKV 标记为下线，然后将其上数据迁移到其它 TiKV 节点，在数据迁移期间 TiKV Pod 依然是 `Running` 状态，数据迁移完成后对应 Pod 才会被删除，缩容时间与待缩容的 TiKV 上的数据量有关，可以通过 `kubectl get tidbcluster -n <namespace> <release-name> -o json | jq '.status.tikv.stores'` 查看 TiKV 是否处于下线（Offline）状态。
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/scale-in-kubernetes.md]:
```suggestion
> - TiKV 组件在缩容过程中会调用 PD 接口将对应 TiKV 标记为下线，然后将其上数据迁移到其它 TiKV 节点，在数据迁移期间 TiKV Pod 依然是 `Running` 状态，数据迁移完成后对应 Pod 才会被删除，缩容时间与待缩容的 TiKV 上的数据量有关，可以通过 `kubectl get tidbcluster -n <namespace> <release-name> -o json | jq '.status.tikv.stores'` 查看 TiKV 是否处于下线 `Offline` 状态。
```

File: dev/tidb-in-kubernetes/deploy/access-tidb.md [hunk 2]
```diff
@@ -23,30 +23,25 @@ tidb:
 在没有 LoadBalancer 时，可选择通过 NodePort 暴露。NodePort 有两种模式：
 
 - `externalTrafficPolicy=Cluster`：集群所有的机器都会给 TiDB 分配 TCP 端口，此为默认值
-- `externalTrafficPolicy=Local`：只有运行 TiDB 的机器会分配 TCP 端口，用于访问本地的 TiDB 实例
 
-    使用 Local 模式时，建议打开 tidb-scheduler 的 `StableScheduling` 特性。tidb-scheduler 会尽可能在升级过程中将新 TiDB 实例调度到原机器，这样集群外的客户端便不需要在 TiDB 重启后更新配置。
+    使用 `Cluster` 模式时，可以通过任意一台机器的 IP 加同一个端口访问 TiDB 服务，如果该机器上没有 TiDB Pod，则相应请求会转发到有 TiDB Pod 的机器上
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/deploy/access-tidb.md]:
```suggestion
    使用 `Cluster` 模式时，可以通过任意一台机器的 IP 加同一个端口访问 TiDB 服务，如果该机器上没有 TiDB Pod，则相应请求会转发到有 TiDB Pod 的机器上。
```

File: v3.0/tidb-in-kubernetes/deploy/access-tidb.md [hunk 3]
```diff
@@ -24,30 +24,25 @@ tidb:
 在没有 LoadBalancer 时，可选择通过 NodePort 暴露。NodePort 有两种模式：
 
 - `externalTrafficPolicy=Cluster`：集群所有的机器都会给 TiDB 分配 TCP 端口，此为默认值
-- `externalTrafficPolicy=Local`：只有运行 TiDB 的机器会分配 TCP 端口，用于访问本地的 TiDB 实例
 
-    使用 Local 模式时，建议打开 tidb-scheduler 的 `StableScheduling` 特性。tidb-scheduler 会尽可能在升级过程中将新 TiDB 实例调度到原机器，这样集群外的客户端便不需要在 TiDB 重启后更新配置。
+    使用 `Cluster` 模式时，可以通过任意一台机器的 IP 加同一个端口访问 TiDB 服务，如果该机器上没有 TiDB Pod，则相应请求会转发到有 TiDB Pod 的机器上
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/deploy/access-tidb.md]:
```suggestion
    使用 `Cluster` 模式时，可以通过任意一台机器的 IP 加同一个端口访问 TiDB 服务，如果该机器上没有 TiDB Pod，则相应请求会转发到有 TiDB Pod 的机器上。
```

File: v3.0/tidb-in-kubernetes/scale-in-kubernetes.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
 > **注意：**
 >
 > - PD、TiKV 组件在滚动升级的过程中不会触发扩缩容操作。
-> - PD、TiKV 组件在缩容过程中都会调用接口下线正在删除的 PD、TiKV 组件在下线时会涉及到数据迁移的操作，所以会消耗比较长的时间。
+> - TiKV 组件在缩容过程中会调用 PD 接口将对应 TiKV 标记为下线，然后将其上数据迁移到其它 TiKV 节点，在数据迁移期间 TiKV Pod 依然是 `Running` 状态，数据迁移完成后对应 Pod 才会被删除，缩容时间与待缩容的 TiKV 上的数据量有关，可以通过 `kubectl get tidbcluster -n <namespace> <release-name> -o json | jq '.status.tikv.stores'` 查看 TiKV 是否处于下线（Offline）状态。
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/scale-in-kubernetes.md]:
```suggestion
> - TiKV 组件在缩容过程中会调用 PD 接口将对应 TiKV 标记为下线，然后将其上数据迁移到其它 TiKV 节点，在数据迁移期间 TiKV Pod 依然是 `Running` 状态，数据迁移完成后对应 Pod 才会被删除，缩容时间与待缩容的 TiKV 上的数据量有关，可以通过 `kubectl get tidbcluster -n <namespace> <release-name> -o json | jq '.status.tikv.stores'` 查看 TiKV 是否处于下线 `Offline` 状态。
```

File: dev/tidb-in-kubernetes/deploy/tidb-operator.md [hunk 5]
```diff
@@ -89,29 +101,33 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
-```
+    > **注意：**
+    >
+    > `<chart-version>` 在后续文档中代表 chart 版本，例如 `v1.0.0`，可以通过 `helm search -l tidb-operator` 查看当前支持的版本。
 
-配置 `/home/tidb/tidb-operator/values-tidb-operator.yaml` 文件中的 `scheduler.kubeSchedulerImage` 为你的 Kubernetes 集群中的镜像。
+2. 配置 TiDB Operator
 
-{{< copyable "shell-regular" >}}
+    TiDB Operator 里面会用到 `k8s.gcr.io/kube-scheduler` 镜像，如果下载不了该镜像，可以修改 `/home/tidb/tidb-operator/values-tidb-operator.yaml` 文件中的 `scheduler.kubeSchedulerImage` 为 `registry.cn-hangzhou.aliyuncs.com/google_containers`。
```
Comment (tennix) [inline dev/tidb-in-kubernetes/deploy/tidb-operator.md]: @lilin90 For English document, this step is unnecessary. Only the first and third steps are needed.

Comment (lilin90) [reply dev/tidb-in-kubernetes/deploy/tidb-operator.md]: @tennix Got it. All other changes apply to the English docs except this step 2?

Comment (tennix) [reply dev/tidb-in-kubernetes/deploy/tidb-operator.md]: Yes.

**Pull request merged**